plugins/woocommerce-admin/client/tasks/fills/experimental-import-products/index.tsx

```diff
@@ -3,18 +3,41 @@
  */
 import { WooOnboardingTask } from '@woocommerce/onboarding';
 import { registerPlugin } from '@wordpress/plugins';
-
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronUp, chevronDown } from '@wordpress/icons';
+import { Button } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { find } from 'lodash';
 /**
  * Internal dependencies
  */
 import CardList from './CardList';
 import { importTypes } from './importTypes';
+import './style.scss';
+import useProductTypeListItems from '../experimental-products/use-product-types-list-items';
+import Stacks from '../experimental-products/stack';
```

Comment:
@moon0326 I found Webpack is showing warnings. 

```
[0] Conflicting order. Following module has been added:
[0]  * css ../../node_modules/.pnpm/css-loader@6.7.1_webpack@5.70.0/node_modules/css-loader/dist/cjs.js!../../node_modules/.pnpm/postcss-loader@3.0.0/node_modules/postcss-loader/src/index.js??ruleSet[1].rules[3].use[2]!../../node_modules/.pnpm/sass-loader@10.2.1_sass@1.49.9+webpack@5.70.0/node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[3].use[3]!./client/tasks/fills/experimental-import-products/cards.scss
[0] despite it was not able to fulfill desired ordering with these modules:
[0]  * css ../../node_modules/.pnpm/css-loader@6.7.1_webpack@5.70.0/node_modules/css-loader/dist/cjs.js!../../node_modules/.pnpm/postcss-loader@3.0.0/node_modules/postcss-loader/src/index.js??ruleSet[1].rules[3].use[2]!../../node_modules/.pnpm/sass-loader@10.2.1_sass@1.49.9+webpack@5.70.0/node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[3].use[3]!./client/tasks/fills/experimental-products/stack.scss
[0]    - couldn't fulfill desired order of chunk group(s)
[0]
[0] webpack 5.70.0 compiled with 1 warning in 10517 ms
[0] No issues found.
[0] assets by status 9.53 MiB [cached] 107 assets
[0] assets by path . 9.13 MiB 116 assets
[0] cached modules 4.44 MiB (javascript) 723 KiB (css/mini-extract) 32.1 KiB (runtime) [cached] 2087 modules
[0]
[0] WARNING in chunk client_tasks_fills_experimental-import-products_CardList_tsx-client_tasks_fills_experimental--c7ef12 [mini-css-extract-plugin]
[0] Conflicting order. Following module has been added:
[0]  * css ../../node_modules/.pnpm/css-loader@6.7.1_webpack@5.70.0/node_modules/css-loader/dist/cjs.js!../../node_modules/.pnpm/postcss-loader@3.0.0/node_modules/postcss-loader/src/index.js??ruleSet[1].rules[3].use[2]!../../node_modules/.pnpm/sass-loader@10.2.1_sass@1.49.9+webpack@5.70.0/node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[3].use[3]!./client/tasks/fills/experimental-import-products/cards.scss
[0] despite it was not able to fulfill desired ordering with these modules:
[0]  * css ../../node_modules/.pnpm/css-loader@6.7.1_webpack@5.70.0/node_modules/css-loader/dist/cjs.js!../../node_modules/.pnpm/postcss-loader@3.0.0/node_modules/postcss-loader/src/index.js??ruleSet[1].rules[3].use[2]!../../node_modules/.pnpm/sass-loader@10.2.1_sass@1.49.9+webpack@5.70.0/node_modules/sass-loader/dist/cjs.js??ruleSet[1].rules[3].use[3]!./client/tasks/fills/experimental-products/stack.scss
[0]    - couldn't fulfill desired order of chunk group(s)
```

I think we could fix this by reordering Stacks/CardList imports.

```
import Stacks from '../experimental-products/stack';
import CardList from './CardList';
import { importTypes } from './importTypes';
import './style.scss';
import useProductTypeListItems from '../experimental-products/use-product-types-list-items';
```

https://stackoverflow.com/questions/60858885/how-do-i-avoid-the-warning-chunk-styles-mini-css-extract-plugin-conflicting-o

> The order needs to be consistent everywhere the imports are referenced; otherwise webpack doesn't know how to order them when it creates the chunks.

Reply:
Thank you for the catch! Updated in https://github.com/woocommerce/woocommerce/pull/32835/commits/d0b500a3f4820ecc2bd94649591749b045d0041d